cmd/controller/run.go

```diff
@@ -17,9 +17,10 @@ import (
 	pkgclient "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/client/clientset/versioned"
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/app"
 	kcclient "github.com/vmware-tanzu/carvel-kapp-controller/pkg/client/clientset/versioned"
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/componentInfo"
```

Comment:
```suggestion
	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/componentinfo"
```

Changing the `I` to `i` since we try to keep all the package names lowercase. This comment will apply to all imports and also the folder names